org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -132,55 +126,51 @@
       }
 
       .crm-accordion-body {
-        padding: 15px 0 !important;
+        padding: 15px 0 0 !important;
 
         .section-shown {
           padding: 0;
         }
       }
     }
 
-    .crm-action-menu .select2-default {      
-      span.select2-chosen {
-        color: $crm-copy;
+    .crm-action-menu .select2-default span.select2-chosen {
+      color: $crm-copy;
 
-        .crm-i {
-          display: none;
-        }
+      .crm-i {
+        display: none;
       }
     } 
 
     .description {
-      margin: 15px;
       font-family: $font-family-base;
+      margin: 15px;      
     }
 
     #_qf_GroupContact_next {
+      background: $brand-primary url(../../../../../i/plus-icon.png) 8px 11px no-repeat;
```

Comment:
Actually, It's used sometime before. It's not recent change. Though, I have updated it and use the image from same extension folder. We can't use font-awesome because it's used for input button and font-awesome will not work for input button.